### PR TITLE
Azure Functions: Inlay Hints with Timer Trigger Cron description #740

### PR DESCRIPTION
**Pull request by maartenba**
image

### CODE REVIEW

**Review by rafaelldi — APPROVED** [review]

**Comment by rafaelldi** [comment]
I will copy it to the new version

**Comment by rafaelldi** [comment]
Here is https://github.com/JetBrains/azure-tools-for-intellij/pull/742

File: PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.Daemon/FunctionApp/Stages/Analysis/TimerTriggerCronExpressionAnalyzer.cs [hunk 1]
```diff
@@ -78,46 +83,71 @@ protected override void Run(IAttribute element, ElementProblemAnalyzerData data,
...
-                CrontabSchedule.Parse(normalizedLiteral, new CrontabSchedule.ParseOptions {IncludingSeconds = true});
+                CrontabSchedule.Parse(normalizedLiteral, new CrontabSchedule.ParseOptions { IncludingSeconds = true });
                 errorMessage = null;
+                try
+                {
+                    description = ExpressionDescriptor.GetDescription(normalizedLiteral);
+                }
+                catch (Exception)
+                {
+                    description = null;
```
Comment (rafaelldi) [inline PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.Daemon/FunctionApp/Stages/Analysis/TimerTriggerCronExpressionAnalyzer.cs:124]: Rider showed me that there may be a null ref exception; so maybe better will be a `""` value.

Comment (maartenba) [reply PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.Daemon/FunctionApp/Stages/Analysis/TimerTriggerCronExpressionAnalyzer.cs:124]: Marked as [CanBeNull] (and using the null reference to skip rendering the inlay hint). Checking once more...

Comment (maartenba) [reply PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.Daemon/FunctionApp/Stages/Analysis/TimerTriggerCronExpressionAnalyzer.cs:124]: Fixed another potential NRE though. Good catch!

**Pull request merged**